default/stringtables/de.txt

```diff
@@ -1901,8 +1901,8 @@ Musikdateien
 OPTIONS_SOUND_FILE
 Klangdateien
 
-OPTIONS_AUTO_EFFECT_DESC
-Automatisch generierte Effektbeschreibungen
+OPTIONS_DUMP_EFFECTS_GROUPS_DESC
+Effektgruppenbeschreibungen anzeigen
```

Comment:
something weird is going on with this pull request. this line is not consistent with the file I have here.